src/5e-SRD-Features.json

```diff
@@ -19,7 +19,110 @@
       "Once you have raged the maximum number of times for your barbarian level, you must finish a Long Rest before you can rage again. You may rage 2 times at 1st level, 3 at 3rd, 4 at 6th, 5 at 12th, and 6 at 17th."
     ],
     "url": "/api/features/rage",
-    "feature_specific": {}
+    "feature_specific": {
+      "actions": [
+        {
+          "name": "Rage",
+          "action_type": "bonus",
+          "duration": "10 turns",
```

Comment:
Actions don't really have durations, effects do. I think having a `duration` attribute on each effect makes more sense than the action itself having a duration.